documentation/weave/index.md

```
@@ -37,52 +37,76 @@ updated: "2024-12-12"
+-   **weave repos verify**: checks whether repos are ready for build (and eventually, whether a pull would produce any conflicts)
+    -   ensure sparse checkout settings are good
+    -   each git inclusion can have `ignore-behind`, `ignore-ahead`, `ignore-divergent`, and `ignore-checkout-consistency`
+-   **weave remap**: transform directory names or filenames (to avoid collisions or for renaming in general)
+-   **weave collisions**: list any potential collisions to console or optionally to a file; optionally/eventually perform custom logic to avoid collisions
+    -   silent options
+-   **weave build**: `repos prepare` and then copy all specified directories and files for active inclusions into dest dir, by inclusion order
+    -   clean: `true` | `false`
+    -   global-copy-strategy: `overwrite` | `no-overwrite` | `skip` | `prompt`
+    -   per-inclusion copy-strategy: `overwrite` | `no-overwrite` | `skip` | `prompt`
+-   **weave watch**: detects changes in active inclusions and copies them to dest
+    -   ?does it
```

Markdownlint (0.37.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)